config/locales/en.yml

```diff
@@ -1085,9 +1085,10 @@ user:
     passwordRequirements: >
       <p>Must be at least 10 characters long <br>
       Must not be the same as your email address<br>
-      A good tip for a secure password is to
-      <a href="https://www.cyberaware.gov.uk/blog/three-random-words">
-      use three random words<span class="u-visually-hidden"> National Cyber Security Centre - External link</span></a>.</p>
+      <a href="https://www.ncsc.gov.uk/collection/top-tips-for-staying-secure-online/use-a-strong-and-separate-password-for-email">
```

Comment:
We've updated this as the previous blog post we were linking to has been removed/archived. This page exists in it's place. It still mentions "three random words" but covers some other general advice too so we've reworded the link text to "Read tips on how to create a strong password" to better reflect the broader content.